include/seqan3/range/container/aligned_allocator.hpp

```diff
@@ -28,6 +28,12 @@ namespace seqan3
  *
  * \details
  *
+ * This class allocates memory at the given alignment boundary. If the specified `alignment` is not supported by the
+ * used allocation method a std::bad_alloc exception will be thrown. For requested alignments larger than
+ * `__STDCPP_DEFAULT_NEW_ALIGNMENT__` the storage will have the alignment specified by the value `alignment`.
+ * Otherwise, the storage is aligned for any object that does not have new-extended alignment and is of the requested
```

Comment:
What is new-extended? Can you link it to some cppreference page?

Reply:
And is the other one not of the requested size?

Reply:
Can you give some examples which "objects" might be effected by the else case?

Reply:
like everything up to `__STDCPP_DEFAULT_NEW_ALIGNMENT__` bytes. IIRC that is 16 byte. But I look into this and extend the example with some comments. I guess it shows this already.